src/app/myProfile/MyProfile.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+import './MyProfile.scss';
+import ProfileRead from './profileRead/ProfileRead';
+import ProfileUpdate, { Profile } from './profileUpdate/ProfileUpdate';
+
+const MyProfile = () => {
+  const [isEditing, setIsEditing] = useState(false);
+  const [profile, setProfile] = useState<Profile>({
```

Comment:
Better to create some external file and import that profile to that file, for readability